app.py

```diff
@@ -31,21 +32,35 @@ def wiki_search():
 @app.route('/heatmap', methods=['GET', 'POST'])
 def calender_heatmap():
     """Returning the plotly visual in html form"""
-    month = request.args.get('month')
-    year = request.args.get('year')
+    month = int(request.args.get('month'))
+    year = int(request.args.get('year'))
 
     # Uses the default values in function if no value is inputted
     if month:
         calendar_heatmap.get_viz(month, year)
     else:
         calendar_heatmap.get_viz()
-    
+
     return render_template('heatmap.html')
-    
 
+# Create route to delete heatmap
 @app.route('/delete_map')
 def delete():
     os.remove('templates/heatmap.html')
+    # os.remove('templates/gauge.html')
```

Comment:
You have this commented-out line further down under a separate endpoint - is there a reason?

Reply:
Oh yeah, I was going to create one route to delete both the heatmap and gauge plot but I figured it'll return an error if one of files is missing. So I created separate routes for each.

Reply:
I fixed it and removed the code line